workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -206,6 +206,12 @@ public static void LaunchLocalDeployment()
             var command = Common.SpatialBinary;
             var commandArgs = "local launch";
 
+            var runtimeIp = GdkToolsConfiguration.GetOrCreateInstance().RuntimeIp;
+            if (!string.IsNullOrEmpty(runtimeIp))
+            {
+                commandArgs = $"{commandArgs} --runtime_ip={runtimeIp}";
```

Comment:
Don't we also need to use the `mobile_launch.json` for this?

Reply:
Correct me if I'm wrong, but mobile_launch.json is only for internal use - we expect external users to use their own config.
That said, allowing to pick launch configuration would be an awesome addition.

Reply:
Oh right, now I remember. This was also one of the downsides of using that: there is no way to define another launch config for mobile if we do that.

Reply:
Lets leave that for later